apps/hash-api/src/graph/ontology/primitive/entity-type.ts

```diff
@@ -50,7 +50,13 @@ export const createEntityType: ImpureGraphFunction<
     title: params.schema.title,
   });
 
-  const schema = { $id: entityTypeId, ...params.schema };
+  const schema = {
+    $schema:
+      "https://blockprotocol.org/types/modules/graph/0.3/schema/entity-type" as const,
+    kind: "entityType" as const,
+    $id: entityTypeId,
+    ...params.schema,
+  };
```

Comment:
For these I think you can just do this
```suggestion
  const schema = {
    $schema:
      "https://blockprotocol.org/types/modules/graph/0.3/schema/entity-type",
    kind: "entityType",
    $id: entityTypeId,
    ...params.schema,
  } as const;
```

If that does work It'd be nice if it was applied elsewhere, it's just a bit cleaner. (Not blocking the PR though)

Reply:
With https://github.com/hashintel/hash/pull/2129/commits/0241c04ec2c6616e744b9b2382a31e18fd9a3e44 it removes the `const` from `$schema`. Do we still want the `const` at the end? I've seen both approaches in the code and found this more clearer.

Reply:
I don't mind, let's just stick with what's already implemented